.gitignore

```diff
@@ -348,3 +348,9 @@ MigrationBackup/
 
 # Ionide (cross platform F# VS Code tools) working folder
 .ionide/
+
+#github folder
+.github
```

Comment:
- [ ] Please let's remove this directory. It's not required to be added here. `.github` directory should be found in your repository to enable github action.

Reply:
Done! =)